docs/src/pages/company/careers/designer.md

```diff
@@ -1,101 +1,101 @@
+- Evolve and refine the Data Grid UX for features such as filtering, column pinning, row editing, and more.
+- Help refine the MUI X documentation, from visual design to copywriting.
+- Support the design team growth by promoting design/product culture, and hiring new members.
 
-### You might fit for this role if you
+### You might fit for this role if you:
 
-- See yourself as a generalist designer, which means you like tackling problems ranging from strategic thinking to execution development (more insight at the recently written [first draft of our design philosophy](https://www.notion.so/Design-philosophy-bdbef60d6fad4134a43870df3e452ebc), available at our public Handbook).
-- Have excellent visual and interaction design chops.
-- Consider yourself a great communicator, both at the written and verbal levels.
-  Designers are key players when it comes to creating alignment and shared understanding, be it from internal to external stakeholders.
+- See yourself as a generalist designer, tackling problems ranging from strategic thinking to execution. (More insight in the [first draft of our design philosophy](https://mui-org.notion.site/Design-philosophy-bdbef60d6fad4134a43870df3e452ebc), available in our public Handbook).
+- Have excellent visual and interaction design skills.
+- Consider yourself a great communicator, both written and verbal.
+  Designers are key players when it comes to creating alignment and shared understanding, whether with internal or external stakeholders.
 - Feel comfortable around code and git (we use GitHub).
-  Developers are MUI's main audience, therefore it's very important to know how to read and experience code for yourself, in order to empathize with our users.
-  You should be capable of tweaking your own JSX and CSS, ensuring that your designs are implemented with the highest fidelity.
-  But don't worry, you'll always have the support of your team.
+  Developers are MUI's main audience, so it's very important to experience our technology for yourself, to empathize with our users.
+  You should be capable of tweaking JSX and CSS, ensuring that your designs are implemented with the highest fidelity.
+  But don't worry, you'll always have the support of your teams' developers.
 
-Previous experience with design systems would be great but not required.
+Previous experience with design systems would be great, but is not required.
 
 ## Benefits & Compensation
 
-We offer competitive compensation depending on the profile and location.
+We offer competitive compensation, aligned with your profile and location.
 We're ready to pay top market rates for a designer that can significantly push the mission forward.
 Other perks are described on [the careers page](/careers/#perks-amp-benefits/).
 
 ## How to apply?
 
-[Apply now for this position 📮](https://airtable.com/shrdqo1Z6srZXGcvh?prefill_Applying+for=Designer).
+[Apply now for this position 📮](https://airtable.com/shrdqo1Z6srZXGcvh?prefill_Applying for=Designer).
```

Comment:
https://airtable.com/shrdqo1Z6srZXGcvh?prefill_Applying+for=Designer

Reply:
👍  @oliviertassinari – Atrefact of the urldecode of the version rescued  from request headers :D